auth0/resource_auth0_connection.go

```diff
@@ -13,6 +13,14 @@ func newConnection() *schema.Resource {
 		Read:   readConnection,
 		Update: updateConnection,
 		Delete: deleteConnection,
+		Importer: &schema.ResourceImporter{
+			State: func(data *schema.ResourceData, i interface{}) ([]*schema.ResourceData, error) {
```

Comment:
For some resources we might be able to use [`ImportStatePassthrough`](https://github.com/hashicorp/terraform/blob/master/helper/schema/resource_importer.go#L50-L52)?

Reply:
Looking at the AWS code, they seem to take the same approach that I did, which was to fully populate the `ResourceData` on a read.

That would make the AWS approach make sense to me